src/assignments/tech-jobs-oo.rst

```diff
@@ -0,0 +1,558 @@
+
+In the ``Job`` class, create a ``ToString()`` method that passes the first test.
+Since the test only checks if the returned string starts and ends with a blank
+line, make that happen.
+
+.. admonition:: Tip
+
+   Do NOT add anything beyond what is needed to make the test pass. You will
+   add the remaining behaviors for ``ToString()`` as you code each new test.
+
+Finish the TDD for ``ToString()``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Code a new test for the second required behavior, then run the tests to make
+   sure the new one fails.
+#. Modify ``ToString()`` to make the new test pass. Also, make sure that your
+   updates still pass all of the old tests.
+#. Continue this test-refactor cycle until all of the behaviors we want for
+   ``ToString()`` work. Remember to add only ONE new test at a time.
+
+Cool! Your ``Job`` class is now complete and operates as desired.
+
+Refactor to DRY the Support Classes
+-----------------------------------
+
+Review the code in the ``Employer``, ``Location``, ``CoreCompetency``, and
+``PositionType`` classes. What similarities do you see?
+
+There is a fair amount of repetition between the classes. As a good coder,
+anytime you find yourself adding identical code in multiple locations you
+should consider how to streamline the process.
+
+   DRY = "Don't Repeat Yourself".
+
+Create a Base Class
+^^^^^^^^^^^^^^^^^^^
+
+Let's move all of the repeated code into a separate class. We will then have
+``Employer``, ``Location``, ``CoreCompetency``, and ``PositionType`` *inherit*
+this common code.
+
+#. Create a new class called ``JobField``.
+#. Consider the following questions to help you decide what code to put in the
+   ``JobField`` class:
+
+   a. What fields do ALL FOUR of the classes have in common?
+   b. Which constructors are the same in ALL FOUR classes?
+   c. Which custom methods are identical in ALL of the classes?
+
+#. In ``JobField``, declare each of the common fields.
+#. Code the constructors.
+#. Add in the custom methods.
+#. Finally, to prevent the creation of a ``JobField`` object, make this class
+   *abstract*.
+
+Extend ``JobField`` into ``Employer``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now that you have the common code located in the ``JobField`` file, we can
+modify the other classes to reference this shared code. Let's begin with
+``Employer``.
+
+#. Modify line 5 to *extend* the ``JobField`` class into ``Employer``.
+#. Next, remove any code in ``Employer`` that matches code from ``JobField``
+   (e.g. the ``Id``, ``Value``, and ``nextId`` fields are shared).
+#. Remove any of the methods that are identical.
+#. The empty constructor is shared, but not the second. Replace the two
+   constructors with the following:
+
+   .. sourcecode:: csharp
+      :lineno-start: 7
+
+      public Employer(string value) : base() 
+      {
+         Value = value;
+      }
+
+   The ``:`` and ``base`` keywords link the ``JobField`` and
+   ``Employer`` classes.
```

Comment:
what i mean here is that this sentence is a little black-boxy - it doesn't really explain what this syntax is -and kind of obscures what ": base" is used for.  Since this will be covered in the inheritance lesson content, what are we adding by keeping this? if you want to keep some explanation of the fact that we need to explicitly inherit the no-arg constructor for the parent class, we should be more detailed in explaining that here. 
describing `:base` as keywords that link two classes is not specific enough to be meaningful

Reply:
I removed the sentence!